src/content/docs/apm/agents/nodejs-agent/troubleshooting/generate-trace-log-troubleshooting-nodejs.mdx

```diff
@@ -13,7 +13,7 @@ redirects:
 Your New Relic [Node.js agent log](/docs/apm/agents/nodejs-agent/installation-configuration/nodejs-agent-configuration/#logging_config) captures errors at the default `info` level. However, when troubleshooting or debugging, generate a more verbose `trace` log to help find and investigate problems.
 
 <Callout variant="important">
-  The `trace` log setting is a highly verbose logging level. To reduce disk space consumption, return the `logging : {` section's `level` to its [original setting](/docs/apm/agents/nodejs-agent/installation-configuration/nodejs-agent-configuration/#logging_config) after testing.
+  The `trace` log setting is a highly verbose logging level. To reduce disk space consumption, return the `logging : {level}` section to its [original setting](/docs/apm/agents/nodejs-agent/installation-configuration/nodejs-agent-configuration/#logging_config) after testing.
 </Callout>
 
 ## Generate log files [#create]
```

Comment:
In line 23, the step about editing the "newrelic.js" file generally does not apply to serverless monitoring. Customers *can* ship an agent in their deployment package (and so also a `newrelic.js` file), but we don't recommend it. Instead, we recommend using the agent contained in the latest Lambda Layer for the Node runtime, which does not use a config file. So for Lambda, the environment variables are the preferred way to set the log level.

Reply:
Thanks!